build/build.go

```diff
@@ -121,8 +121,8 @@ func importWithSrcDir(path string, srcDir string, mode build.ImportMode, install
 
 // ImportDir is like Import but processes the Go package found in the named
 // directory.
-func ImportDir(dir string, mode build.ImportMode) (*PackageData, error) {
-	pkg, err := build.ImportDir(dir, mode)
+func ImportDir(dir string, mode build.ImportMode, installSuffix string, buildTags []string) (*PackageData, error) {
+	pkg, err := NewBuildContext(installSuffix, buildTags).ImportDir(dir, mode)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Okay, so this ended up being not as big of a change as I originally thought. I thought you'd have to copy all of [this logic](https://github.com/gopherjs/gopherjs/blob/45518c130e5bd1525f20110830a4986365a153de/build/build.go#L70-L112) too. But I see now that you don't. That logic was not a part of `ImportDir` before, but it's only needed for building the standard Go library with GopherJS. That means `build.ImportDir` can't be used to build stdlib packages. Oh well.